model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/custom/CustomCreateIndexChange.java

```diff
@@ -74,7 +74,7 @@ public SqlStatement[] generateStatements(Database database) {
         }
         try {
             // To check that the table already exists or not on which the index will be created.
-            if (!SnapshotGeneratorFactory.getInstance()
+            if (getTableName() == null || !SnapshotGeneratorFactory.getInstance()
```

Comment:
It may happen that table is not yet set in scenario when [`generateStatements`](https://github.com/keycloak/keycloak/blob/58131f1dccb7113cb708407d6cf90842216c8d00/model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/custom/CustomCreateIndexChange.java#L60) method is called from [`AbstractChange.supports`](https://github.com/liquibase/liquibase/blob/ae4f651d241c45f2d09d4c23d6da8a56446a6f17/liquibase-standard/src/main/java/liquibase/change/AbstractChange.java#L387) which is the case since this [change](https://github.com/liquibase/liquibase/pull/4476/files#diff-402d22cfe1bfa96a8a1ef2bc2210864d96082666359af99c266575781f40da89R109).